javascripts/_bootstrap-carousel.js

```diff
@@ -1,223 +0,0 @@
-/* ========================================================================
```

Comment:
I can't seem to find this file elsewhere. Is it not needed any longer?

Reply:
We have added the 'bootstrap.js' which contains combination of all bootstrap related js files.

Reply:
Ok, thx